components/annot-explanation.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div>
+    <b-card no-body class="mb-1">
+      <b-card-header header-tag="header" class="p-1" role="tab">
+        <b-button block v-b-toggle.accordion-explanation variant="info">{{ card_title }}</b-button>
+      </b-card-header>
+      <b-collapse id="accordion-explanation" accordion="my-accordion" role="tabpanel">
+        <b-card-body>
+          <b-card-text>{{ explanation }}</b-card-text>
+        </b-card-body>
+      </b-collapse>
+    </b-card>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "sub-explanation",
```

Comment:
Should we be providing name fields for all components or just pages? This question, of course, addresses all the components in this commit. But regarding this one -

Why is the `annot-explanation` component's name just 'sub-explanation'? If we're giving names to components,
how can we make consistent naming, and with regard to `annot-explanation` bring that naming in line with
the names for the other annotation page components?

Reply:
Good point. This was an oversight. I am not sure what uses the `name` attribute here. But it would make sense to me to have `name` = `fileName`. Do you agree?

Reply:
Let's just do some quick research on this `name` property. It was recommended in the Nuxt tutorials for pages -from what I recall.

Reply:
https://v2.vuejs.org/v2/api/?redirect=true#name

> #name
> 
> Allow the component to recursively invoke itself in its template. Note that when a component is registered globally with Vue.component(), the global ID is automatically set as its name.
> 
> Another benefit of specifying a name option is debugging. Named components result in more helpful warning messages. Also, when inspecting an app in the [vue-devtools](https://github.com/vuejs/vue-devtools), unnamed components will show up as <AnonymousComponent>, which isn’t very informative. By providing the name option, you will get a much more informative component tree.

Reply:
And it is an option for the linter: https://eslint.vuejs.org/rules/require-name-property.html

Reply:
`name` = `filename` works. I don't see a compelling reason for something more descriptive than that.

Reply:
`name` = `filename` works. I don't see a compelling reason for something more descriptive than that.

Reply:
OK, will add these then!